packages/crawler/tests/e2e/helpers/puppeteer-helper.ts

```diff
@@ -0,0 +1,121 @@
+import type { Page, WaitForOptions } from 'puppeteer';
+import { getLoaderScript } from '@porsche-design-system/components-js/partials';
+
+export type PdsTestingContext = {
+  bodyHtml: string;
+  firstPdsVersionPrefixes?: string[];
+  secondPdsVersionPrefixes?: string[];
+};
+export const getInternalLoaderScriptForPrefixes = (prefixes: string[]): string =>
+  prefixes.reduce((result, prefix) => result + getLoaderScript({ prefix: prefix }), '');
+
+export const getExternalLoaderScriptForPrefixes = (prefixes: string[]): string => {
+  const prefixesLoaders = prefixes.reduce(
+    (result, prefix) =>
+      result +
+      `
+    porscheDesignSystem.load(\{ prefix: '${prefix}' \});
+  `,
+    ''
+  );
+  // the script below has been copied from https://designsystem.porsche.com/v2/
+  return `
```

Comment:
Perhaps we can get rid of the formatting. Might be related to prettier which could be disabled for a section or next line.

Reply:
Fixed